### ISSUE TITLE
Support Dark mode for all Launchpad UI

### ISSUE THREAD

**Issue by vedmant** [cypress-io/cypress#29153]
### What would you like?

Dark mode right now is widespread and I think any proper application should support it. Cypress however doesn't have a dark mode. Having system wide dark theme and flashing white when everything is dark is quite uncomfortable. 

Would be also a nice feature to switch browser mode white testing and setup default theme in config.

### Why is this needed?

- Dark mode of general interface
- Switching light/dark for tests

**Comment by MikeMcC399** [comment]
@vedmant 

https://github.com/dingraham/cypress-runner-themes from @dingraham may be helpful for you.

**Comment by jennifer-shehane** [comment]
The main interface is dark in 10.0+ versions. 

![Screenshot 2024-03-18 at 10 55 37 AM](https://github.com/cypress-io/cypress/assets/1271364/54b21495-8dbc-4344-9501-e05e49aa4c51)

Are you referring to the other pages, wanting those to be in dark mode? I'll be honest, I don't see this work being prioritized. 

![Screenshot 2024-03-18 at 10 55 04 AM](https://github.com/cypress-io/cypress/assets/1271364/fb25528b-9090-439f-a975-0ffad01abd9a)
![Screenshot 2024-03-18 at 10 55 11 AM](https://github.com/cypress-io/cypress/assets/1271364/eca79f4e-ffbb-4744-8c36-91d69937cd52)
![Screenshot 2024-03-18 at 10 55 20 AM](https://github.com/cypress-io/cypress/assets/1271364/4bb942f1-b8e6-4476-ad5c-3dd22d89e685)

**Comment by vedmant** [comment]
@jennifer-shehane Yes, other pages, plus main interface top bar (on right side), it's still white. 

Also I think right now it's a single theme for both light and dark system preferences, dark sidebar is dark for light theme as well (which I've seen some people complained about it). I think it just makes sense to support two theme light and dark, nowadays it's a must to have proper dark theme, IMHO.

**Comment by ELigoP** [comment]
When it's evening, and all your open windows support dark mode, and specs list is shining at you, it is preferable to be in dark mode too. But I like Cypress anyway :)

**Comment by davidmatium** [comment]
@ELigoP  checkout my plugin in the meantime, will help a bit with that!
https://github.com/dingraham/cypress-runner-themes

**Comment by muchbetterug** [comment]
I like cypress really to, but darkmode IS important.

the switching between a dark code editor and the light cypress ui drives me crazy at night

i hope the cypress-runner-themes works, thx!

**Comment by Pachat** [comment]
I like so much Cypress. And used it years ago. 
Now I can only work in Dark mode. Any way to achieve that?
Until that, i'll have to Playwright.

**Comment by alexross93** [comment]
Dark mode across the board would be really great and should be out of the box, the initial light theme Welcome page is migraine-fuel for us dark mode users.

**Comment by wpiron10** [comment]
Hi, I really like Cypress but I really think that in 2026, approximately 2 years after this issue was opened, this feature is a must.
Most people working on Cypress spend hours on it, several days a week. The need for the dark mode IS important in this case, it is not just a shiny color theme, it is useful for the users that USE your library. Dark mode is implemented almost everywhere, the question of whether to put in the effort to implement it or not should be asked again. Nothing fancy is needed, just to start somewhere.
